src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackSubmitPageUiTest.java

```diff
@@ -146,6 +146,7 @@ private void testSubmitAction() throws Exception {
         assertNull(BackDoor.getFeedbackResponse(
                                fqConstSum.getId(), "IFSubmitUiT.instr@gmail.tmt", "IFSubmitUiT.instr@gmail.tmt"));
 
+        submitPage.uncheckSendConfirmationEmailCheckbox();
         submitPage.clickSubmitButton();
```

Comment:
This uncheck-the-box-then-click-the-submit-button routine is repeated everywhere. Abstract it to the `FeedbackSubmitPage` class like this:
```java
public void submitWithoutConfirmationEmail() {
    uncheckSendConfirmationEmailCheckbox();
    clickSubmitButton();
}
```